go/host/node/node.go

```diff
@@ -315,18 +315,19 @@ func (a *Node) ReceiveTx(tx common.EncryptedTx) {
 	a.txP2PCh <- tx
 }
 
-// CreateSubscription sets up a subscription between the host and the enclave.
-func (a *Node) CreateSubscription(encryptedLogSubscription common.EncryptedLogSubscription) error {
```

Comment:
I had to move the generation of the UUID to the API side, so that I know which sub ID to use for unsubscribe calls.